README.md

```diff
@@ -327,3 +327,18 @@ where you extracted the archive, run the configure script using `./configure` an
 then build the programs using the command "make".  The command line programs will
 be built in the "src" directory.
 
+### CMAKE
+
+In case of modern usage. A Cmake version is provided by yixuanzhou@sjtu.edu.cn. 
+
+```sh
+cd spheretree && mkdir build
+cd build && cmake .. -DCMAKE_BUILD_TYPE=Release -DENABLE_SANITIZER=OFF
```

Comment:
I tried to configure with `ENABLE_SANITIZER=ON`, but get an error. Perhaps include the Sanitizer module from the `cmake/` directory? I think it would be nice the either fix this (best) or set `ENABLE_SANITIZER=OFF` by default (worse). Also, please consider removing unused `cmake/*.cmake` files.

Reply:
Sure, I'll remove unused ones, and make SANITIZER work correctly. Sorry for my careless inspection.